### DISCUSSION TITLE
Environment variables of Dwains 2.0 in Dwains 3.0

### DISCUSSION THREAD

**Discussion by MikeTheCleric** [dwainscheeren/dwains-lovelace-dashboard#573] [Q&A]
Hi! I've used for a lot Dwains Dashboard 2.0 and I've modified using also the variables generated by yaml files like this one (rooms.yaml):

![image](https://user-images.githubusercontent.com/52752310/173355176-129fb7da-e78e-4ef6-b82c-eeb5558de340.png)

Is it possible to recreate a system like this in Dwains 3.0? I heavily use this configurations in the previous theme and I don't want to throw away my modifications?
If not possible the Dwains 2.0 will be mantained (to adapt new HA core version or so on) for a while or not? So I can use it without the risk of an HA update that make it useless.
Thanks in advice!

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
